Classes/Public/Manager/OCTManagerConstants.h

```diff
@@ -44,17 +45,29 @@ typedef NS_ENUM(NSInteger, OCTMessageFileType) {
 
 typedef NS_ENUM(NSInteger, OCTMessageCallEvent) {
     /**
-     * Call dialing
+     * Call was answered.
      */
-    OCTMessageCallEventDial,
+    OCTMessageCallEventAnswered,
 
     /**
-     * Call was missed.
+     * Call was unanswered.
      */
-    OCTMessageCallEventMissed,
+    OCTMessageCallEventUnanswered,
+};
+
+typedef NS_ENUM(NSInteger, OCTCallStatus) {
+    /**
+     * Call is currently ringing.
+     */
+    OCTCallStatusRinging,
+
+    /**
+     * Call is currently dialing a chat.
+     */
+    OCTCallStatusDialing,
 
     /**
-     * Call has ended.
+     * Call is currently active in session.
      */
-    OCTMessageCallEventEnd,
+    OCTCallStatusActive,
```

Comment:
Oh, I see. Sorry, I was confused with all this call stuff :grinning:  Okay, let it be like that, at least for now.